docs/tutorials/tutorial_params_options_simulate.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>Some comments:</p><p> </p><ul><li>second paragraph: "In&nbsp;<code>respy</code>&nbsp;the istantaneous"</li><li>when the terms of the formulas are explained, include $t$. "experience in period t", shock in period t</li><li>"Normal distribution" to "normal ..."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>